<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabLogQuery</class>
 <widget class="QWidget" name="TabLogQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_targets">
     <property name="title">
      <string>目标设备</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_targets">
      <item>
       <widget class="QLabel" name="label_ipListHint">
        <property name="text">
         <string>IP 列表（每行一个）：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="txt_logIPList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
         </font>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::LineWrapMode::NoWrap</enum>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_filters">
     <property name="title">
      <string>查询条件</string>
     </property>
     <layout class="QFormLayout" name="formLayout_filters">
      <item row="0" column="0">
       <widget class="QLabel" name="label_logPath">
        <property name="text">
         <string>日志路径：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txt_logPath">
        <property name="text">
         <string>/apps/m580cn/log</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_keyword">
        <property name="text">
         <string>关键字（可选）：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txt_keyword"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_timeRange">
        <property name="text">
         <string>时间范围（可选）：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_time">
        <item>
         <widget class="QDateTimeEdit" name="dt_start">
          <property name="displayFormat">
           <string>yyyy-MM-dd HH:mm</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_to">
          <property name="text">
           <string>至</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dt_end">
          <property name="displayFormat">
           <string>yyyy-MM-dd HH:mm</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_preview">
     <property name="title">
      <string>日志预览（双击设备可单独查看）</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_preview">
      <item>
       <widget class="QTreeWidget" name="tree_logResults">
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
        </property>
        <property name="headerHidden">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>设备 IP</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>文件名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>文件大小</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>修改时间</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>下载进度</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_actions">
     <item>
      <widget class="QPushButton" name="btn_queryLogs">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #FF9800; color: white; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>查询日志</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_downloadSelected">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>下载选中日志</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_downloadAll">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>下载全部日志</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
